packages/keyring-api/src/btc/types.ts

```diff
@@ -39,18 +16,18 @@ export enum BtcMethod {
   SendBitcoin = 'sendBitcoin',
 }
 
-export const BtcP2wpkhAccountStruct = object({
+export const BtcAccountStruct = object({
   ...KeyringAccountStruct.schema,
 
   /**
    * Account address.
    */
-  address: BtcP2wpkhAddressStruct,
+  address: string(),
```

Comment:
I think we should still validate the address formats here. We could use `bitcoin-address-validation` package for this, however, given that we are using 1 single struct, it will be hard to make sure the `address` is valid for the given `type`.

So maybe we should have 4 different structs instead (similarly to `BtcP2wpkhAccountStruct`, we would have `BtcP2shAccountStruct`, etc...), allowing us to strictly check the `address` is valid for the account's `type`?

WDYT?

Reply:
The address should not be validated because for the extension, the address is a `string`. For example if the Snap tells you that your address is `fooBar` then that's your address. you can't decide that you don't agree with the Snap.
Knowing the address format is not needed in the business logic.

Reply:
After discussing, all types were added.